tests/Shared/TempDirectory.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+public sealed class TempDirectory : IDisposable
+{
+    public string Path { get; } = Directory.CreateTempSubdirectory(".aspire-tests").FullName;
```

Comment:
I would have kept `FullName`, but it's not important if you don't agree.

Reply:
Yeah, I prefer `Path`